pkg/utils/logger.go

```diff
@@ -0,0 +1,62 @@
+package utils
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"k8s.io/klog/v2"
+)
+
+// LoggerFromContext returns logr.Logger implementation retrieved from
+// ctx sent by caller. If no logger was set in ctx then a default
+// logr.Logger implementation is returned.
+func LoggerFromContext(ctx context.Context) logr.Logger {
+	return klog.FromContext(ctx)
+}
+
+// LoggerWithName returns a new logger instance with name appended to
+// logger.
+func LoggerWithName(logger logr.Logger, name string) logr.Logger {
+	return klog.LoggerWithName(logger, name)
+}
+
+// LoggerWithValues returns a new logger instance with key-value pairs (kvs)
+func LoggerWithValues(logger logr.Logger, kvs ...interface{}) logr.Logger {
+	return klog.LoggerWithValues(logger, kvs...)
+}
```

Comment:
I don't see any benefit in using this functions than directly using the corresponding klog functions.

Reply:
Removed wrapper functions to klog 46b3a74c6be88506877eecd3ab7f4c1d9a2aa566